misc/dist/deb-scripts/postinst

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
My hero!

(But seriously -- does Debian not ship with /bin/bash in the base system?)

Reply:
And if not, have you tried this script on whatever they _do_ ship, just to make sure it works?

Reply:
I think Bash is still in the base system, but for a while now /bin/sh is Dash, not Bash, which is faster and more POSIX compliant and therefore the preference for init scripts and such. And yep, it works!